homeassistant/components/dlink/entity.py

```diff
@@ -24,10 +25,6 @@ def __init__(
         """Initialize a D-Link Power Plug entity."""
         self.data = data
         self.entity_description = description
-        if config_entry.source == SOURCE_IMPORT:
```

Comment:
I suggest keeping this case as long as we keep the import flow around. It can be removed when we remove the import flow. Otherwise it's a breaking change.

Reply:
Well spotted 👍 :(

Reply:
In that case, we need to revert the PR. As we cannot implement new naming styles + keeping this around.

Reply:
Revert: #86936